content/legal/acronym_guidelines/index.md

```diff
@@ -0,0 +1,130 @@
+---
+title: "Oracle/JCP Acronym Guidelines"
+date: 2019-06-26T16:10:38-04:00
+description: "Guidelines for the use of existing Oracle/JCP Acronyms.  Supplement to the Eclipse Foundation Guidelines for Eclipse Logos & Trademarks Policy for Oracle/JCP Acronyms"
+hide_sidebar: "false"
+---
+
+<p><em>Guidelines for the use of existing Oracle/JCP Acronyms.  Supplement to the Eclipse
+Foundation <a href="https://jakarta.ee/legal/trademark_guidelines/">Guidelines for Eclipse Logos &
+Trademarks Policy</a> for Oracle/JCP Acronyms</em></p>
```

Comment:
The "for Oracle/JCP Acronyms" at the end looks like it's part of the name of the supplement.
Really it just duplicates the first sentence.  Please remove it.

Reply:
Updated.